src/DIRAC/ConfigurationSystem/Agent/RucioSynchronizerAgent.py

```diff
@@ -322,26 +509,35 @@ def execute(self):
           except Duplicate:
             pass
           except Exception as err:
-            self.log.error('Cannot create identity %s for account %s : %s', dn, account, str(err))
+            self.log.error(
+                "Cannot create identity for account", "[DN: %s, account: %s, Error: %s]" %
+                (dn, account, str(err)))
```

Comment:
Different collaboration will have different ways to define the scopes

Reply:
Maybe. See my comment below. Config file-less approach is to use a simple algorithm with a scope being a string at a defined position in the LFN. Belle approach is different, but then you have your scope alg hardwired in Rucio code and you needs a config file. I thing if we need more complicated scope alas we should make an effort to define them in Dirac CS.